<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTeacherForm_template</class>
 <widget class="QDialog" name="AddTeacherForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of dialog">Add teacher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="teacherNameLabel">
       <property name="text">
        <string>Teacher:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teacherNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="morningsAfternoonsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Mornings-afternoons behavior:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="exclusiveRadioButton">
          <property name="text">
           <string>Exclusive (Morocco)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneDayExceptionRadioButton">
          <property name="text">
           <string>One day exception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="twoDaysExceptionRadioButton">
          <property name="text">
           <string>Two days exception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="threeDaysExceptionRadioButton">
          <property name="text">
           <string>Three days exception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fourDaysExceptionRadioButton">
          <property name="text">
           <string>Four days exception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fiveDaysExceptionRadioButton">
          <property name="text">
           <string>Five days exception</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="unrestrictedRadioButton">
          <property name="text">
           <string>Unrestricted (Algeria)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addTeacherPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
